root/static/scripts/edit/URLCleanup.js

```diff
@@ -757,7 +757,13 @@ const CLEANUPS = {
         if (!/^(images|www)$/.test(subdomain)) {
           switch (id) {
             case LINK_TYPES.mailorder.artist:
-              return {result: product === undefined};
+              return {
+                error: l(
+                  `Please link to the main page for the artist,
+                   not a specific product.`,
+                ),
+                result: product === undefined,
+              };
```

Comment:
`error` should not be set when `result` is `true`.

Reply:
I thought it wouldn't matter since we should only be loading the error when result is false :)

Reply:
So you'd rather have this set so that it checks whether the check is false, and then return either one way or the other? Well then. It honestly seems like a waste of lines to me, but if it makes you happy :D